newIDE/app/src/MainFrame/index.js

```diff
@@ -92,8 +92,8 @@ import OpenFromStorageProviderDialog from '../ProjectsStorage/OpenFromStoragePro
 import SaveToStorageProviderDialog from '../ProjectsStorage/SaveToStorageProviderDialog';
 import OpenConfirmDialog from '../ProjectsStorage/OpenConfirmDialog';
 import verifyProjectContent from '../ProjectsStorage/ProjectContentChecker';
-const GD_STARTUP_TIMES = global.GD_STARTUP_TIMES || [];
 
+const GD_STARTUP_TIMES = global.GD_STARTUP_TIMES || [];
```

Comment:
Avoid having unrelated or useless changes in your PR

Reply:
Oh! I will keep that in mind.